.babelrc

```diff
@@ -9,6 +9,7 @@
             }
         ],
         "@babel/preset-react",
+        "@babel/preset-typescript"
```

Comment:
Can you split this PR to two: 1) changes for removing model interface; 2) typescript, please? We'll take care of the 2) later.

Reply:
New PR (only JS) here: https://github.com/Exabyte-io/mode.js/pull/35